vnext/ReactUWP/Views/MenuFlyoutItemViewManager.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "MenuFlyoutItemViewManager.h"
+#include <Views/ShadowNodeBase.h>
+#include "TouchEventHandler.h"
+#include "ViewPanel.h"
+#include <Utils/ValueUtils.h>
+
+#include <Modules/NativeUIManager.h>
+#include <Utils/PropertyHandlerUtils.h>
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+namespace winrt {
+  using namespace Windows::UI::Xaml::Controls::Primitives;
+  using namespace Windows::UI::Xaml::Interop;
+}
+
+
+namespace react {
+  namespace uwp {
```

Comment:
sync with master, then 'yarn install' and 'yarn format'. it will help you format all cpp files.